validation/genesis/genesis-predeploy_test.go

```diff
@@ -26,7 +28,23 @@ func perChainTestName(chain *superchain.ChainConfig) string {
 	return chain.Name + fmt.Sprintf(" (%d)", chain.ChainID)
 }
 
+var temporaryOptimismDir string
+
 func TestGenesisPredeploys(t *testing.T) {
+	//
+	// Clone optimism into gitignored temporary directory (if that directory does not yet exist)
```

Comment:
Perhaps also note that the optimism repo is cloned into a directory that is at the same level as `superchain-registry` with some context (dep resolution errors). I imagine someone else will walk into the same trap as us in the future if they try to modify this 😬